spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.11/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala211DataprocImage13AcceptanceTest.java

```diff
@@ -18,7 +18,9 @@
 import java.util.Collections;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 
+@Ignore
 public class Scala211DataprocImage13AcceptanceTest extends DataprocAcceptanceTestBase {
```

Comment:
Let's delete this file and other such files instead.

Reply:
David suggested to keep these files for now.